common/src/main/java/com/wynntils/screens/guides/ingredient/GuideIngredientItemStackButton.java

```diff
@@ -68,13 +69,28 @@ public void renderButton(PoseStack poseStack, int mouseX, int mouseY, float part
         }
     }
 
+    // FIXME: This should be painted by ItemHighlightFeature instead...
+    private CustomColor getHighlightColor(int tier) {
+        return switch (tier) {
+            case 0 -> new CustomColor(102, 102, 102);
+            case 1 -> new CustomColor(255, 247, 153);
+            case 2 -> new CustomColor(255, 255, 0);
+            case 3 -> new CustomColor(230, 77, 0);
```

Comment:
Is this not the same memory leak concept I asked about you earlier? Can we make this static?

Reply:
(not leak..., but high allocation, since this is called render time.)

Reply:
Yes I can, but see also the comment above:
```
 // FIXME: This should be painted by ItemHighlightFeature instead...
```

I'm not really happy about this, but did not have stamina to continue fighting with this code. I'd rather see the guide item stacks being drawn as normal item stacks, so our item stack features can do their job. Now we duplicate that code, and we lose any settings the user has made for these colors. 

But I'll make them static for now, it might be a lousy design but there's no need for it do have lousy memory footprint as well.